project/src/main/webapp/events.html

```diff
@@ -53,11 +53,19 @@
                   <input placeholder="Event Name" name="name" type="text" class="validate" autocomplete="off" spellcheck="false" maxlength="15"> 
                   <input type="text" name="location" placeholder="Enter a Location" class="validate" autocomplete="off" spellcheck="false">
                   <div class="row">
-                    <div class="col s6">
-                      <input type="text" placeholder="Event Date" class="datepicker" name="date" id="datepicker" style="font-color: white">
+                    <div class="col s5">
+                      <input type="text" placeholder="Date" class="datepicker" name="date" id="datepicker" style="font-color: white">
                     </div>
-                    <div class="col s6">
-                      <input type="text" placeholder="Event Time" class="timepicker" name="time" id="timepicker" style="font-color: white">
+                    <div class="col s3">
+                      <input type="text" placeholder="Time" class="timepicker" name="time" id="timepicker" style="font-color: white">
+                    </div>
+                    <div class="col s4">
+                      <select name="capacity" id="capacity" class="validate">
+                        <option value="" disabled selected>Capacity</option>
+                        <option value="15">15</option>
+                        <option value="50">50</option>
+                        <option value="100">100</option>
+                      </select>
```

Comment:
If you want, a range option like [this](https://materializecss.com/range.html)
```
  <form action="#">
    <p class="range-field">
      <input type="range" id="test5" min="0" max="100" />
    </p>
  </form>
```
would be pretty useful here!

Reply:
We can use this! I just kept it as buckets for now because I thought it would be easier to manage in datastore but I will make this update soon.

Reply:
Up to you, just a cool feature I usually like seeing on other websites.